config/config.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"time"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+// TODO: remove hello world example config option
+// Config represents service configuration for dp-frontend-release-calendar
+type Config struct {
+	BindAddr                   string        `envconfig:"BIND_ADDR"`
+	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
+	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
+	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	HelloWorldEmphasise        bool          `envconfig:"HELLO_WORLD_EMPHASISE"`
+}
+
+var cfg *Config
+
+// Get returns the default config with any modifications through environment
+// variables
+func Get() (*Config, error) {
+	if cfg != nil {
+		return cfg, nil
+	}
+
+	cfg = &Config{
+		BindAddr:                   ":27700",
```

Comment:
Please update https://github.com/ONSdigital/dp/blob/main/guides/PORTS.md to refer to the new name of this service for this port

Reply:
https://github.com/ONSdigital/dp/pull/248